Tests/BrowserServicesKitTests/Autofill/AutofillEncrypterTests.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  AutofillEncrypterTests.swift
+//  DuckDuckGo
+//
+//  Copyright © 2021 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import XCTest
+import CryptoKit
+@testable import BrowserServicesKit
+
+class AutofillEncrypterTests: XCTestCase {
+
+    func testWhenMessageIsEncrypted_ThenCanBeDecryptedUsingAuthenticationData() throws {
+        let key: [UInt8] = SymmetricKey(size: .bits256).withUnsafeBytes { $0.map { $0 }}
+        let iv: [UInt8] = SymmetricKey(size: .bits256).withUnsafeBytes { $0.map { $0 }}
```

Comment:
You could also do this here, purely a matter of personal preference though 🙂 

```suggestion
        let key: [UInt8] = SymmetricKey(size: .bits256).withUnsafeBytes { Array($0) }
        let iv: [UInt8] = SymmetricKey(size: .bits256).withUnsafeBytes { Array($0) }
```